pycbc/waveform/ringdown.py

```diff
@@ -537,7 +790,8 @@ def multimode_base(input_params, domain, freq_tau_approximant=False):
                             input_params['f_lower'],
                             input_params['f_final'],
                             l=int(lmn[0]), m=int(lmn[1]),
-                            inclination=input_params['inclination'])
+                            inclination=input_params['inclination'],
+                            pol=pols[lmn])
             outplus.data += hplus.data
             outcross.data += hcross.data
     else:
```

Comment:
Other to do: Add documentation for new parameters (`ref_amp`, `dbeta`, `dphi`, etc.) to the various `get_` functions that are outward facing, but refer them to `td_damped_sinusoid` for the math.

Reply:
Done